resq_tools/lib/screens/resistance_screen.dart

```diff
@@ -171,27 +170,26 @@ class _ResistanceScreenState extends State<ResistanceScreen> {
             },
           ),
           const SizedBox(height: 12),
-          DropdownMenu<UndergroundType>(
-            width: double.infinity,
-            initialSelection: _measurementConfig.undergroundType,
-            onSelected: (value) {
-              setState(() {
-                _measurementConfig = _measurementConfig.copyWith(
-                  undergroundType: value,
-                );
-              });
-              _updateMeasurementConfig(context);
-            },
-            dropdownMenuEntries:
-                UndergroundType.values
-                    .map(
-                      (undergroundType) => DropdownMenuEntry(
-                        value: undergroundType,
-                        label:
-                            undergroundType.getTranslation(context.l10n) ?? '',
-                      ),
-                    )
-                    .toList(),
+
+          _buildDropdownCard(
+            context: context,
+            title: '''
+${_measurementConfig.undergroundType.getTranslation(context.l10n)}''',
```

Comment:
remove apostrophes

Reply:
Needed since otherwise the 80 char limit is exceeded 🥸

Reply:
you could make it like that

```suggestion
            title: _measurementConfig
                              .undergroundType
                              .getTranslation(context.l10n) ?? '',
```

Reply:
I like to use string interpolation, since if the localization is not available it will display "null" - which I think makes it easier to detect compared to an empty string. But can be changed if needed